src/redis-cli.c

```diff
@@ -1807,6 +1819,7 @@ static void usage(int err) {
 "  -n <db>            Database number.\n"
 "  -3                 Start session in RESP3 protocol mode.\n"
 "  -x                 Read last argument from STDIN.\n"
+"  -X                 Read <tag> argument from STDIN.\n"
```

Comment:
please add a short example to the examples below (it's a bit hard to understand that tagging concept without an example.